### ISSUE TITLE
Can't Save Processing Settings

### ISSUE THREAD

**Issue by alex-phillips** [Receipt-Wrangler/receipt-wrangler-api#312]
I've set up receipt wrangler per the documentation with docker, but when I go to settings to attempt to set up receipt processing settings, I fill out the form but when I hit save, nothing happens. No error, no network activity in the browser network tab. What am I missing?  Using the latest instance published as of today.

**Comment by alex-phillips** [comment]
![image](https://github.com/user-attachments/assets/af0b9abb-f664-4e9a-8480-5cad2eb1f41b)

Also nothing in the container logs that seem like an issue.

**Comment by Noah231515** [comment]
@alex-phillips do you mind posting a gif/video of the issue? Im not able to reproduce this. Do you have any other prompts or receipt processing settings set up? Also what type of database are you using (postgresql, sqlite, MariaDB/mysql)?

**Comment by alex-phillips** [comment]
https://github.com/user-attachments/assets/2f644707-6a33-4f0e-aed8-a53ef17688a0

Database is mariadb. This is a fresh setup, this is the first thing I tried doing. Below is my docker-compose:
```yaml
version: '3'
services:
  receiptwrangler:
    container_name: receiptwrangler
    image: noah231515/receipt-wrangler:latest
    entrypoint: ./entrypoint.sh
    restart: unless-stopped
    volumes:
      - ${APPDATA}/receiptwrangler:/app/receipt-wrangler-api/data
    environment:
      - ENCRYPTION_KEY=REMOVED
      - DB_USER=REMOVED
      - DB_PASSWORD=REMOVED
      - DB_NAME=REMOVED
      - DB_HOST=REMOVED
      - DB_PORT=3306
      - DB_ENGINE=mariadb
      - SECRET_KEY=REMOVED
    ports:
      - 9082:80
```

**Comment by Noah231515** [comment]
@alex-phillips Thanks for all of this information. 

I see what the issue is now, a combination of things that lead to this.
The issue is is that the prompt field is really asking to select a configured prompt, see https://receiptwrangler.io/docs/concepts/system-settings/prompts.

Then you will be able to select that prompt in the autocomplete and proceed.

The documentation does not state that a prompt needs to be set up first, I will update the docs to make that clear and additionally there is no error of any kind that indicates that the list is empty for a required field.

In summary: Set up a prompt first, then you will be able to select it from the dropdown.

**Comment by alex-phillips** [comment]
Can you give an example of a prompt? I'm a bit confused on what it should be especially since it's allowing me to select variables to use in it.
